CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+x.y.z Release notes (yyyy-MM-dd)
+=============================================================
+### Enhancements
+* Add support for Xcode 11 Beta 1. Xcode betas are only supported when building
+  from source, and not when using a prebuilt framework.
+  ([PR #6164](https://github.com/realm/realm-cocoa/pull/6164)).
+
+### Fixed
+* <How to hit and notice issue? what was the impact?> ([#????](https://github.com/realm/realm-js/issues/????), since v?.?.?)
```

Comment:
```suggestion
```

Reply:
You were the one that added these lines to the new changelog section template. Do you no longer want them there?

Reply:
I usually delete them after having updated the changelog. That could also be done just before releasing though.

Reply:
I don't really see the point in adding something to the file then immediately removing it the first time it's edited. Either it's useful to have it there for people making edits (and thus should remain until the final edit for release) or it isn't useful and shouldn't be there at all.

Reply:
Yeah, I agree we should just delete at release time - I assumed this was going to be released without waiting for anything else.